src/ConfigCreator.php

```diff
@@ -254,6 +254,12 @@ public function fixMissingConfiguration(): array {
     /** @var array $missing_configurations */
     $missing_configurations = [];
 
+    // Hack in a media.type.image.
+    // @todo create dependencies of missing configuration.
+    $missing_config_name = ConfigName::createByFullName('media.type.image');
+    $config_importer = $this->configImporterPluginManager->createInstance($missing_config_name->getType());
+    $config_importer->importConfig('', $missing_config_name->getFullName());
```

Comment:
I don't think this is sufficient for our intendet tests, see comment in src/Plugin/TestsiteBuilder/ConfigImporter/EntityBrowser.php as well.